docs/pages/blog/lab-pickers-to-mui-x.md

```diff
@@ -0,0 +1,63 @@
+---
+title: The date pickers are now part of the Advanced components
+description: XXXX
+date: 2022-04-03T00:00:00.000Z
+authors: ['flaviendelangle']
+tags: ['MUI X', 'News']
+---
+
+We're excited to announce that, after more than 18 month in the lab, the date picker components are now part of the X packages.
+
+## What is MUI X ?
+
+Since its creation, MUI have always kept at heart to provide great components for free.
+However, some projects require several full-time developers and can not be sustainable relying only on open-source income.
+To be able to keep improving our products, including the open source one, we decided in 2020 to launch the _Advanced components_, a new set of components available under two licenses:
+
+- MUI X: the MIT licensed, free to use version
+- MUI X Pro: the commercially licensed version with the most advanced features
+
+The Advanced component launched by our team have been the Data Grid, with the `@mui/x-data-grid` and `@mui/x-data-grid-pro` packages.
+More than one year later, this new source of income has allowed our company to hire several new developers and to increase the amount of work achieved, even on the open-source team.
+
+## Why move the date pickers to MUI
```

Comment:
I assume you mean
```suggestion
## Why move the date pickers to MUI X
```

But the question you answer below is
```suggestion
## Will I have to pay?
```

Reply:
I think this section could have some additional contain about why is the picker a good candidate for X.